tests/fixtures/gtfs2ntfs/minimal/output/companies.txt

```diff
@@ -1,2 +1,2 @@
 company_id,company_name,company_address,company_url,company_mail,company_phone
-default_agency_id,mon agence,,http://kisio.org,,
+mon_agence,mon agence,,http://kisio.org,,
```

Comment:
I don't like you change that. We should keep the test of default agency_id

by the way should the defautl `agency_id` be `default_agency_id` or `1` like in the specs https://github.com/CanalTP/transit_model/blob/master/src/documentation/gtfs_to_ntfs_specs.md#reading-agencytxt?

@papailio @prhod

Reply:
according to the specs, the default value of `agency_id` should be `1`

Reply:
ok so we should change the function `default_agency_id` and the fixtures.

Reply:
Done in c950dca47798fbbb7c76fb3ca8b00475dae31e4d.